proxy_docker/app/script/walletoperations.sh

```diff
@@ -62,6 +62,44 @@ spend() {
   return ${returncode}
 }
 
+bumpfee() {
+  trace "Entering bumpfee()..."
+
+  local request=${1}
+  local txid=$(echo "${request}" | jq ".txid" | tr -d '"')
```

Comment:
```bash
jq -r 
```
will output the txnid string without double quotes i believe. (no need for tr)

Reply:
Oh man, I have a ton of `jq ".whatever" | tr -d '"'` in cyphernode!  lol!  Thanks, I will clean the whole project.

Reply:
haha love it